osquery/events/darwin/kernel_util.cpp

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
Create a similar implementation file that does nothing in the `void loadKernelExtension();` within the linux/etc directories. Let them be blank as long as they aren't used/needed. Then we have a similar auto-load/check for crash "flow" across platforms.